commands/http/handler.go

```diff
@@ -106,20 +105,18 @@ func (i internalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	ctx, cancel := context.WithCancel(node.Context())
-	defer cancel()
-	/*
-		TODO(cryptix): the next line looks very fishy to me..
-		It looks like the the context for the command request beeing prepared here is shared across all incoming requests..
```

Comment:
is this not the case? i was suspecting this was part of the http handler failure we had been seeing